### PR TITLE
Check for MacStudio Target Value

### PR DESCRIPTION
**Pull request by PhaneeshB**
Need to check the vulkan target for MacStudio

### CODE REVIEW

**Comment by pashu123** [comment]
`black --line-length=79 .`

**Review by powderluv — DISMISSED** [review]
modulo linting

**Comment by PhaneeshB** [comment]
linting error resolved

File: .github/workflows/test-models.yml [hunk 1]
```diff
@@ -30,8 +30,6 @@ jobs:
             suite: cpu
           - os: MacStudio
             suite: gpu
-          - os: MacStudio
-            suite: vulkan
```
Comment (powderluv) [inline .github/workflows/test-models.yml:34]: i would switch this to cpu so we don't run on Mac CPU anylonger. But it can be a follow on commit

**Review by powderluv — APPROVED** [review]

**Pull request merged**